config/google_analytics.yml

```diff
@@ -0,0 +1,2 @@
+global: 'UA-50123418'
+education_benefits: 'UA-50123418-7'
```

Comment:
These are both 'prod' keys - one that's for the global vets.gov account, another that's just for the education benefits form. I initially had these as environment variables in application.yml, but started thinking that a distinct config file made more sense given the possible number of tokens involved in the future. There is also a test/sandbox GA environment that could be the default one, and then the tokens in this PR are written down in the overlay..